down.sh

```diff
@@ -6,9 +6,12 @@ then
     exit
 fi
 
+DEFAULT_WORKING_DIR="."
+WORKING_DIR="${GITHUB_ACTION_PATH:-$DEFAULT_WORKING_DIR}"
```

Comment:
instead of directly using `GITHUB_ACTION_PATH` can we make this more general to this repo, and then just set that env var in the consuming action? 

ie, `TESTNET_TERRAFORM_WORKING_DIR` or... something

Reply:
`GITHUB_ACTION_PATH` is the env variable set automatically by github in the runner :sweat_smile:

Reply:
Still, we could rename it on the runner (or inface it'd just be passed in anyway?). Just seems odd to tie this repo to GHA so much